data/core/command.lua

```diff
@@ -98,12 +141,25 @@ local function perform(name, ...)
 end
 
 
+---Performs a command.
+---
+---The arguments passed into this function is forwarded to the predicate function. </br>
+---If the predicate function returns more than 1 value, the second to last values
+---are passed to the function associated to the command.
+---
+---Otherwise, the the arguments passed into this function is passed directly
+---to the function associated with the command.
+---@see core.command.predicate
+---@see core.command.predicate_function
+---@param ... any
+---@return boolean # true if the command is performed successfully.
 function command.perform(...)
   local ok, res = core.try(perform, ...)
   return not ok or res
 end
```

Comment:
I'd explicitly add the `name` parameter to this, to make it more clear.
```suggestion
function command.perform(name, ...)
  local ok, res = core.try(perform, name, ...)
  return not ok or res
end
```

Reply:
This is unrelated to the PR since it only concerns documentation.

Reply:
yeah, I misunderstood; this makes sense.